packages/store/internals/src/memoize.ts

```diff
@@ -28,7 +28,7 @@ function areArgumentsShallowlyEqual(
  *
  * @ignore
  */
-export function memoize<T extends (...args: any[]) => any>(
+export function ɵmemoize<T extends (...args: any[]) => any>(
```

Comment:
Shouldn't this be aliased with the barred-o in the `internals` package exports as opposed to internally within the package?

Reply:
I think we could make all of the namings without barred-o and re-export them with `ɵ`, but this package is 'exports-only', which means it only exports the stuff which is used outside. As thus I'm guessing this would've been an extra code.